visitor.php

```diff
@@ -88,7 +89,7 @@ public function format(): array
                     '%s %s%s',
                     $this->tag,
                     $this->type,
-                    ($this->name !== null) ? (' $' . $this->name) : ''
+                    $this->name !== null ? " \$$this->name" : ''
```

Comment:
Please use sprintf or wrap variables in `"{$var}"`

Reply:
If you mix string literals with variable viktor cannot figure out what is going on (👁️ strain)

Reply:
:ok_hand: